src/dbnode/storage/series/buffer.go

```diff
@@ -545,8 +543,22 @@ func (b *dbBufferBucket) finalize() {
 	b.resetBootstrapped()
 }
 
+func (b *dbBufferBucket) empty() bool {
```

Comment:
you think this will be ok perf-wise? I assume thats why we had the bool to begin with

Reply:
Yeah this should be fine, we don't actually call it from any high frequency code paths.  We used to have a high frequency call site with it, although now it was refactored out.